pkg/deploy/api/validation/validation.go

```diff
@@ -148,6 +148,23 @@ func ValidateDeploymentConfigStatusUpdate(newConfig *deployapi.DeploymentConfig,
 func ValidateDeploymentConfigRollback(rollback *deployapi.DeploymentConfigRollback) field.ErrorList {
 	result := field.ErrorList{}
 
+	if len(rollback.Name) == 0 {
+		result = append(result, field.Required(field.NewPath("name"), "name of the deployment config is missing"))
+	} else if !kvalidation.IsDNS1123Subdomain(rollback.Name) {
+		result = append(result, field.Invalid(field.NewPath("name"), rollback.Name, "name of the deployment config is invalid"))
+	}
+
+	specPath := field.NewPath("spec")
+	if rollback.Spec.Revision < 0 {
```

Comment:
You cannot rollback a config with latestVersion=0 because it is still undeployed. You can specify revision=0 (it's actually the default in `oc`) and we will roll back to the previous version from the one you are on.